_docs/payout-details.md

```diff
@@ -423,3 +439,100 @@ BJ
 <div class="alert alert-info" markdown="1">
 **Note** `XOF::Bank` payouts are currently in beta phase. At this time, we offer payouts to accounts in Senegal and Benin only.
 </div>
+
+# SouthAfrica
+
+## ZAR::Bank
+
+For South African bank payments please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "street": "Main Street",
+  "postal_code": "AB0001",
+  "city": "Cape Town",
+  "bank_code": "334810",
+  "bank_account": "12345678",
+  "phone_number": "+27119785313",
+  "transfer_reason_code": "185"
```

Comment:
since we already have `reason` for MAD::Cash we might want to use the same field?